### ISSUE TITLE
Error When Attempting To Inject

### ISSUE THREAD

**Issue by Atsoup** [Kaldaien/SpecialK#2]
I recently had to reinstall my Windows 10, and my Special K (Version 41) does not inject. I moved the install file to my C:\ and installed it. Upon installation, I attempted to push the inject button for Special K, but it gives me the following error: Special K32 is not a valid win32 application. I tried the following:
- Disabled Windows Defender.
- Made exclusions on Malware Bytes.
- Turned off Secure Boot State (And verified it).
- Installed Windows Developer Tools to see if that would change anything.
- Used different compatibility options until the properties settings.
- And finally, updated Windows to the creator's update (Also checked for old registry keys that were recommended in the unofficial guide of Special K).

**Comment by Idearum** [comment]
Sounds like the download somehow got corrupt. Download the latest SpecialK archive manually and throw the contents of the package into My Documents\My Mods\SpecialK and then run SKIM64.exe from that same folder and try again.

**Comment by Atsoup** [comment]
I have done what you suggest Idearum, but it did not solve my problem. The error message remains.

**Comment by Idearum** [comment]
Yeah, sorry, there's not a lot more I can suggest. That error message is Windows saying that it either a) corrupt, b) doesn't have all the required prerequisites to run the application, or c) something is blocking it from executing (AVs, EMET, etc).

It's basically the least informative error message Windows can throw in the whole world.

**Comment by Atsoup** [comment]
I have fixed the issue. My issue was that the installation of 'Visual c++ Redistributable 2017 x86' was messed up. I fixed my issue by using this program (https://support.microsoft.com/en-us/help/17588/fix-problems-that-block-programs-from-being-installed-or-removed), and I uninstalled all of Visual 2015 redistributable packages (x64, x86). I attempted to uninstall the 2017 ones, but they failed to uninstall each time. I restarted my computer and installed the 2017 Visual C++ Redistributables (x64, x86), and Special K began working again.

TL;DR:
1. Download this program: https://support.microsoft.com/en-us/help/17588/fix-problems-that-block-programs-from-being-installed-or-removed
2. Uninstall all Visual C++ Redistributables from 2015 and up.
3. Restart computer.
4. Install the Visual C++ Redistributables from https://download.microsoft.com/download/6/A/A/6AA4EDFF-645B-48C5-81CC-ED5963AEAD48/vc_redist.x86.exe
5. Hooray! I hope it worked for you!
